extensions/ql-vscode/src/remote-queries/analyses-results-manager.ts

```diff
@@ -151,6 +162,27 @@ export class AnalysesResultsManager {
     void publishResults([...resultsForQuery]);
   }
 
+
+  public async loadDownloadedArtifacts(
+    allAnalysesToCheck: AnalysisSummary[]
+  ) {
+    const allDownloadedAnalyses = await asyncFilter(allAnalysesToCheck, x => this.isDownloadedNotInMemory(x));
```

Comment:
Is this a name change only that you are suggesting? If so, would it be better to have `isAnalysisDownloadedNotInMemory`?

Reply:
I'm suggesting removing the part that it checks if the analysis is already in memory, since that is already happening by `loadAnalysesResults` which is being called by this function. And once we remove that part I think the name can be simplified too.

Reply:
So `isAnalysisDownloadedNotInMemory` changes to:

```ts
private async isAnalysisDownloaded(analysis: AnalysisSummary): Promise<boolean> {
    return await fs.pathExists(createDownloadPath(this.storagePath, analysis.downloadLink));
  }
```

and here the code changes to

```ts
const allDownloadedAnalyses = await asyncFilter(allAnalysesToCheck, x => this.isAnalysisDownloaded(x));
```

Reply:
Ah...yes. That is simpler.